index.html

```diff
@@ -1,315 +1,357 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles.css">
-  <link rel="preconnect" href="https://fonts.gstatic.com">
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet" />
-  <title>Portfolio</title>
-</head>
-<body>
-  <section class="headersec">
-    <div class="topnav-logo">
-      <h4>My Logo</h4>
-    </div>
-    <div class="menu-icon-wrapper">
-      <div class="menu-icon" onclick="toggleCssMenu(this)">
-        <div class="three-line">
-          <span></span>
-          <span></span>
-          <span></span>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header id="header">
+      <div class="header-container">
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
-      </div>
-    </div>
-    <div id="cssmenu">
-
-      <ul>
-        <li><a href="#">Contact</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Potfolio</a></li>
-      </ul>
-    </div>
-    <div class="about-flex-container">
```

Comment:
Thank you for adding background images on mobile view. But they are not the same as in the design and their position is also not matching with the design. Please see the design page for mobile and make corrections accordingly. Moreover, an image is overlapping icons on the large view. Kindly see the below screenshot for reference.
![overlap](https://user-images.githubusercontent.com/35504149/130114768-69e11b7e-084f-4767-b176-70b7f99242d3.png)